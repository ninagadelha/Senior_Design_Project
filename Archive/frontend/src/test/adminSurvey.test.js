import renderer from "react-test-renderer";
import AdminSurvey from "../routes/adminSurvey";

jest.mock("../components/AdminNavBar");
jest.mock("../components/footer");
jest.mock("../styles/adminEditQuestions.css");
jest.mock("react-router-dom");
jest.mock("axios");

const renderTree = (tree) => renderer.create(tree);
describe("<AdminSurvey>", () => {
  it("should render component", () => {
    expect(renderTree(<AdminSurvey />).toJSON()).toMatchSnapshot();
  });
});
