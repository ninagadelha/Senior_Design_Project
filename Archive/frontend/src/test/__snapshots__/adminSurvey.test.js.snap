// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AdminSurvey> should render component 1`] = `
<div
  className="admin-survey-container"
>
  <div
    style={
      {
        "marginTop": "20px",
      }
    }
  >
    <label>
      Select a Survey to Edit: 
      <select
        onChange={[Function]}
      >
        <option
          disabled={false}
          value=""
        >
          Please select a survey name
        </option>
      </select>
    </label>
  </div>
  <div
    className="survey-questions"
  >
    <div>
      <h1>
        Edit Survey Questions
      </h1>
    </div>
  </div>
</div>
`;
